src/Algebra/Graph/Acyclic/AdjacencyMap.hs

```diff
@@ -0,0 +1,449 @@
+module Algebra.Graph.Acyclic.AdjacencyMap (
+  -- * Data structure
+  AdjacencyMap, aam,
+
+  -- * Basic graph construction primitives
+  empty, vertex, disjointOverlay, disjointConnect, vertices,
+
+  -- * Graph properties
+  isEmpty, edgeList, vertexList, edgeSet, vertexSet,
```

Comment:
Please stick to the same export order as in other modules, for consistency of the documentation: 

https://hackage.haskell.org/package/algebraic-graphs-0.4/docs/Algebra-Graph-AdjacencyMap.html#g:4